RockPaperScissorGameBot/Cards/CardsFactory.cs

```diff
@@ -0,0 +1,112 @@
+﻿using AdaptiveCards;
+using Microsoft.Bot.Schema;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace RockPaperScissorGameBot.Cards
+{
+    public class CardsFactory
+    {
+        public Attachment CreateThankYouCardAttachment(string memberName)
+        {
+            var card = new HeroCard
+            {
+                Title = "Rock, Paper, Scissor Game",
+                Text = $"Thank You {memberName} for playing. We will send you the scorecard when the game is over."
```

Comment:
I'd include their selection in the message, so user knows that we recorded their choice correctly. Also useful if they later forget what they picked :)